package.json

```diff
@@ -10,13 +10,12 @@
     "plugins",
     "environment",
     "env",
-    "dotenv",
-    "runtime"
```

Comment:
Maybe we can still have `"runtime"` as a keyword and add `"build"` as a new keyword?